AndroidManifest.xml

```diff
@@ -125,6 +125,12 @@
             android:exported="false"
             android:label="@string/search_users_and_groups_hint" />
 
+        <provider
+            android:name=".ui.adapter.DiskLruImageCacheFileProvider"
```

Comment:
I'm sorry, you already commented it in the description: "it can be used to send via whatsapp, or as a file to another app".  

I apologize.